src/main/groovy/edu/oregonstate/mist/metaxe/XEAppsResource.groovy

```diff
@@ -0,0 +1,139 @@
+package edu.oregonstate.mist.metaxe
```

Comment:
Sure. What do you think should be logged? Keep in mind that exceptions are already logged by  GenericExceptionMapper.